src/config.c

```diff
@@ -3167,6 +3167,7 @@ standardConfig static_configs[] = {
     createBoolConfig("lazyfree-lazy-user-del", NULL, DEBUG_CONFIG | MODIFIABLE_CONFIG, server.lazyfree_lazy_user_del, 1, NULL, NULL),
     createBoolConfig("lazyfree-lazy-user-flush", NULL, DEBUG_CONFIG | MODIFIABLE_CONFIG, server.lazyfree_lazy_user_flush, 1, NULL, NULL),
     createBoolConfig("repl-disable-tcp-nodelay", NULL, MODIFIABLE_CONFIG, server.repl_disable_tcp_nodelay, 0, NULL, NULL),
+    createBoolConfig("repl-mptcp", NULL, IMMUTABLE_CONFIG, server.repl_mptcp, 0, isValidMptcp, NULL),
```

Comment:
Does this config really need to be immutable?

Reply:
I suppose it's immutable. there is no reason to change it dynamically.

Reply:
If we make it mutable and users change it when replication is already connected, it will not have any effect. Only if the replica needs to reconnect some time later it will be able to use MPTCP, but this may never happen. Maybe that can be confusing to users.

If there is a bad network and there are many reconnects, users can enable MTPCP to fix the problem?

I'm just guessing pros and cons of mutable vs immutable. I'm fine with keeping it immutable.